File: trunk/acn/acn-trabalho-final-net/acn-trabalho-final/dominio/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace acn_trabalho_final.dominio
{
    public class Pessoa
    {
        public int Codigo { get; set; }
        public String Nome { get; set; }
        public int Id { get; set; }

        public Pessoa() { }
        public Pessoa(int Id, int Codigo, String Nome) {
            this.Id = Id;
            this.Codigo = Codigo;
            this.Nome = Nome;
        }
    }
}

File: acn/acn-appengine-gwt/src/com/puc/acn/client/dominio/Pessoa.java
package com.puc.acn.client.dominio;

public class Pessoa extends BaseBean {

	private static final long serialVersionUID = 1L;

	private int codigo;
	
	private String nome;
	
	public Pessoa() {
	}
	
	public Pessoa (int id, int codigo, String nome) {
		this.setId(id);
		this.codigo = codigo;
		this.nome = nome;
	}
	
	public int getCodigo() {
		return codigo;
	}
	
	public void setCodigo(int codigo) {
		this.codigo = codigo;
	}
	
	public String getNome() {
		return nome;
	}
	
	public void setNome(String nome) {
		this.nome = nome;
	}
}
